System/mas.1d.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# <bitbar.title>Mac App Store</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>iosdeveloper</bitbar.author>
+# <bitbar.author.github>iosdeveloper</bitbar.author.github>
+# <bitbar.desc>Lists available updates from the Mac App Store using mas-cli (https://github.com/argon/mas). Based on brew-updates.1h.sh</bitbar.desc>
+# <bitbar.image>http://i.imgur.com/yPeXDAj.png</bitbar.image>
+# <bitbar.abouturl>https://github.com/matryer/bitbar-plugins/blob/master/System/mas.1d.sh</bitbar.abouturl>
```

Comment:
Looks like this has a dependency on `mas` - which is not listed in the metadata. Could you please add it? Thanks